src/auth/test/auth.service.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { UnauthorizedException } from '@nestjs/common';
```

Comment:
I strongly recommend that you move all your test scripts to the `test` folder instead of place them in the `src` one, that will make your folder scaffolding cleaner and easier to understand